### PR TITLE
Update Faucet to use updated Tx builder from Agora

### PR DESCRIPTION
**Pull request by hewison-chris**
A refactor to use the updated method for setting up the TxBuilder from Agora.

### CODE REVIEW

File: source/faucet/main.d [hunk 1]
```diff
@@ -68,7 +68,7 @@ import vibe.inet.url;
 import vibe.web.rest;
 
 /// The keys that will be used for generating transactions
-private SecretKey[PublicKey] secret_keys;
+private KeyPair[PublicKey] allKeys;
```
Comment (linked0) [inline source/faucet/main.d:71]:
```suggestion
private KeyPair[PublicKey] all_keys;
```
Is there any reason to use the name instead of `allKeys`?

Comment (hewison-chris) [reply source/faucet/main.d]: No I just copied it from the example usage in the TXBuilder documentation. I have renamed to `known_keys` which is correct style and a better name.

**Comment by hewison-chris** [comment]
This will need a rebase once Agora PR https://github.com/bosagora/agora/pull/3256 is merged.

**Review by linked0 — APPROVED** [review]
LGTM

**Pull request merged**